app/views/restaurants/edit.html.haml

```diff
@@ -0,0 +1,27 @@
+= form_for(@restaurant, url: restaurant_path, html: {method: :patch}) do |f|
```

Comment:
Do you really need a new form with all these settings. Is there a way to stay DRY and extract the form to a partial and reuse ut for both `new` and `edit` actions? Or to extract the fields?